### ISSUE TITLE
Links broken and Requirements/Results unclear

### ISSUE THREAD

**Issue by MekWarrior** [vanvalenlab/intro-to-deepcell#36]
In `pretrained_models` section, we have a broken link. Item 2 in the 'Generating predictions with DeepCell.org' section. The offending link is **the available models and data formatting requirements**

3 other points of concern specific to Mesmer:
1) We do not specify resolution of the data in magnification or um per pixel. This is a key factor in the accuracy of predictions and should be called out specifically (a code snippet to help facilitate people rescaling would be ideal)
2) We should call out the option of submitting a blank channel of data if they only have a nuclear stain/marker or vice versa (a small code snippet would be helpful here as well)
3) We do not specify what the "results" are, so in the case of single-channel they are left to guess which is whole-cell and which is nuc. We should call this out explicitly.

**Comment by bioinfbloke** [comment]
I tried submitting one of our 2 channel nuc/cyto images to https://deepcell.org/predict and am confused about why the results seemed to return a mask but didn’t delineate the boundaries between the cytoplasm (all the cells are merged/overlapping). I notice from your example this is also the case so I am clear on how someone would use this output for further analysis.

**Comment by PingjunChen** [comment]
I am trying to use https://deepcell.org/predict. I'm confused about how to save a 2-channel image. May I ask for your suggestion signs on how to prepare the image for prediction? @MekWarrior @bioinfbloke

**Comment by PingjunChen** [comment]
> I tried submitting one of our 2 channel nuc/cyto images to https://deepcell.org/predict and am confused about why the results seemed to return a mask but didn’t delineate the boundaries between the cytoplasm (all the cells are merged/overlapping). I notice from your example this is also the case so I am clear on how someone would use this output for further analysis.

The output results (***_feature_0/***_feature_1) look like normal images. But loading them into to program (e.g., scikimage.io.imread), you will find that there are channel first images with float data type owing distinct values of each whole-cell/nuclear.  This demo https://github.com/angelolab/ark-analysis/blob/master/templates/Segment_Image_Data.ipynb helps a lot to understand the whole-cell/nuclear segmentation process.

**Comment by ngreenwald** [comment]
Sorry about that, didn't have my notifications turned on for this repo. Thanks for the explanation Pingjun, let me know if either of you have any remaining questions

**Comment by PingjunChen** [comment]
Thanks for your kind reply. @ngreenwald Right now one thing I'm not quite sure about is the image normalization for the two-channel inputs. The data I have is IMC data with the mcd format, and I export the two single-channel images (one for nuclear and one for membrane) from the MCD Viewer.  The maximum intensity of these images is larger than 255 and in float format. Right now the way I do image normalization is just to divide the maximum of both channel images separately to make sure their range to be between 0.0 to 1.0. I'm not sure is the right way to conduct image normalization to obtain the optimal performance. 

For the current results I obtain, the whole-cell looks promising, but the nucleus results seem missing a lot. That's the reason I doubt the manner I do image normalization might be not right.

**Comment by ngreenwald** [comment]
Can you post an example image?

**Comment by PingjunChen** [comment]
I mainly follow the ipython notebook [Segment_Image_Data.ipynb](https://github.com/angelolab/ark-analysis/blob/master/templates/Segment_Image_Data.ipynb ) to perform the whole-cell/nuclear segmentation. I got the whole cell segmentation from **segmentation_labels.loc[fov_to_display + "_feature_1", :, :, 'whole_cell'].values**, and the nuclear segmneation from **segmentation_labels.loc[fov_to_display, :, :, 'nuclear'].values**. The examplar results are shown below. Suggestions would be greatly appreciated @ngreenwald 

## Whole-cell segmentation, looks promising to me
![ROI001_ROI_001_whole_cell](https://user-images.githubusercontent.com/32883033/119743246-12151f00-be4f-11eb-9257-ce82e6581b69.png)

## Nuclear segmentation, missing quite a lot
![ROI001_ROI_001_nuclear](https://user-images.githubusercontent.com/32883033/119743352-4ee11600-be4f-11eb-9052-cfaee1a07630.png)

**Comment by ngreenwald** [comment]
Huh, that is bizarre, haven't seen that before. Can you zip the raw image and attach it?

**Comment by PingjunChen** [comment]
[ROI001.zip](https://github.com/vanvalenlab/intro-to-deepcell/files/6550191/ROI001.zip)
@ngreenwald  Demo images are attached. I took the 191Ir_191Ir as the nuclear channel, and the 176Yb_NaKATPase as the membrane channel.

**Comment by ngreenwald** [comment]
@PingjunChen can you try re-running the command with `scale` set to 2? The IMC data is around 1um/pixel, and our training data is 0.5um per pixel 

https://github.com/angelolab/ark-analysis/blob/9043668cfc5700363b42b59a8b0c831117f1a158/ark/utils/deepcell_service_utils.py#L16

**Comment by PingjunChen** [comment]
@ngreenwald Thanks a lot for your information and suggestions. Resetting the scale to 2.0 greatly improves the whole-cell/nuclear segmentation performance. 
## Whole-cell segmentation is more precise (fine-grained) compared to before (previously multiple cells can be segmented as one)
![ROI001_ROI_001_whole_cell](https://user-images.githubusercontent.com/32883033/120018715-28d18800-bfad-11eb-93a8-76d7ed60580d.png)
## Nuclear segmentation looks consistently with whole-cell segmetnation now
![ROI001_ROI_001_nuclear](https://user-images.githubusercontent.com/32883033/120018906-61716180-bfad-11eb-88fb-46a45cd3c4b8.png)
